src/annotator/frame-observer.js

```diff
@@ -53,7 +53,7 @@ export class FrameObserver {
   async _addFrame(frame) {
     this._annotatableFrames.add(frame);
     try {
-      await onDocumentReady(frame);
+      await onNextDocumentReady(frame);
```

Comment:
To recap the PR description: `onDocumentReady` is now a function that takes a callback which can be invoked multiple times. `onNextDocumentReady` is the equivalent of the _old_ `onDocumentReady` function that returns a promise, and thus only resolves once.